core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -121,7 +121,7 @@ protected[controller] object RestApiCommons {
  */
 protected[controller] trait RespondWithHeaders extends Directives {
     val allowOrigin = `Access-Control-Allow-Origin`(AllOrigins)
-    val allowHeaders = `Access-Control-Allow-Headers`("Authorization", "Content-Type")
+    val allowHeaders = `Access-Control-Allow-Headers`("Authorization", "Content-Type", "User-Agent")
```

Comment:
what is the value of the controller accepting the user-agent?
should nginx just drop this header?